tests/unit/test_home.py

```diff
@@ -1,48 +1,92 @@
 
 
 def describe_translate_moisture():
     def test_translate_moisture_25(mocker):
         number = mocker.patch("db.fetch", return_value=200)
         returned_result = translate_moisture(25)
         assert "Moisture levels critical." in returned_result
+        assert "12%" in returned_result
 
     def test_translate_moisture_75(mocker):
-        number = mocker.patch("db.fetch", return_value=200)
+        number = mocker.patch("db.fetch", return_value=213)
         returned_result = translate_moisture(75)
         assert "Moisture levels balanced." in returned_result
+        assert "35%" in returned_result
 
     def test_translate_moisture_125(mocker):
-        number = mocker.patch("db.fetch", return_value=200)
+        number = mocker.patch("db.fetch", return_value=226)
         returned_result = translate_moisture(125)
         assert "Moisture within acceptable parameters." in returned_result
+        assert "55%" in returned_result
 
     def test_translate_moisture_175(mocker):
-        number = mocker.patch("db.fetch", return_value=200)
+        number = mocker.patch("db.fetch", return_value=378)
         returned_result = translate_moisture(175)
         assert "Moisture decreasing." in returned_result
+        assert "46%" in returned_result
 
     def test_translate_moisture_225(mocker):
-        number = mocker.patch("db.fetch", return_value=200)
+        number = mocker.patch("db.fetch", return_value=9999)
         returned_result = translate_moisture(225)
         assert "Warning: Dry conditions detected." in returned_result
+        assert "2%" in returned_result
 
     def test_translate_moisture_275(mocker):
-        number = mocker.patch("db.fetch", return_value=200)
+        number = mocker.patch("db.fetch", return_value=13)
         returned_result = translate_moisture(275)
         assert "Alert: Severe dehydration likely." in returned_result
+        assert "2115%" in returned_result
 
     def test_translate_moisture_325(mocker):
-        number = mocker.patch("db.fetch", return_value=200)
+        number = mocker.patch("db.fetch", return_value=325)
         returned_result = translate_moisture(325)
         assert "CRITICAL STATUS!" in returned_result
+        assert "100%" in returned_result
 
     def test_translate_moisture_out_of_bounds(mocker):
-        number = mocker.patch("db.fetch", return_value=200)
+        number = mocker.patch("db.fetch", return_value=3)
         returned_result = translate_moisture(9999)
         assert "Reading outside expected parameters" in returned_result
+        assert "%" not in returned_result
 
     def test_translate_moisture_string(mocker):
         number = mocker.patch("db.fetch", return_value=200)
         returned_result = translate_moisture("ABCD")
         assert "Non numberic values provided. Consult administrator" in returned_result
+        assert "%" not in returned_result
+
+
+def test_show_homepage(monkeypatch):
+    app = create_app()
```

Comment:
May not need app at all, since you can get client as a fixture.

If you still needed it you can "pull" app as a fixture too